stacks-signer/src/config.rs

```diff
@@ -228,13 +228,12 @@ impl TryFrom<RawConfigFile> for Config {
                 raw_data.endpoint.clone(),
             ))?;
 
-        let stackerdb_contract_id =
-            QualifiedContractIdentifier::parse(&raw_data.stackerdb_contract_id).map_err(|_| {
-                ConfigError::BadField(
-                    "stackerdb_contract_id".to_string(),
-                    raw_data.stackerdb_contract_id,
-                )
-            })?;
+        let stackerdb_contract_id = match raw_data.stackerdb_contract_id {
+            Some(id) => QualifiedContractIdentifier::parse(&id).map_err(|_| {
+                ConfigError::BadField("stackerdb_contract_id".to_string(), id.clone())
+            })?,
+            None => boot_code_id("signers", raw_data.network == Network::Mainnet),
```

Comment:
Is it an issue to run the signer against testnet?

Reply:
Nope, this is just setting the boot code address depending on the signer's config